tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php

```diff
@@ -1044,7 +1070,7 @@ public function pass($a, $b) {
     public function testFix71($expected, $input = null)
     {
         $this->fixer->configure([
-            'elements' => ['method', 'const'],
+            'elements' => ['method' => ClassAttributesSeparationFixer::SPACING_ONE, 'const' => ClassAttributesSeparationFixer::SPACING_ONE],
```

Comment:
Now all tests use the new config format so the old format is not covered by tests anymore. Please add a test that uses the old format and asserts that:
* it has the same behavior as before;
* a deprecation notice is triggered.

Reply:
Added https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4755